libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts

```diff
@@ -72,10 +75,12 @@ const abandonSeedAddresses: Record<string, string> = {
  * DO NOT USE AS RECIPIENT OR SIGN TRANSACTIONS INTO THEM
  * @param {*} currencyId
  */
+// TODO: signature should be  getAbandonSeedAddress(currencyId: CryptoCurrencyId)
 export const getAbandonSeedAddress = (currencyId: string): string => {
   invariant(
     abandonSeedAddresses[currencyId] !== undefined,
     `No abandonseed available for ${currencyId}`
   );
-  return abandonSeedAddresses[currencyId];
+
+  return abandonSeedAddresses[currencyId]!;
```

Comment:
I don't know this syntax, what it does ? 
the ! at the end ?

Reply:
The `!` says to typescript "don't worry this value is never null or undefined".
I can attest it because of the `invariant` call before. Ideally we should do as I precised in the todo and change the signature from string to CryptoCurrencyId. I started this change and it impacted way too many files for this PR so I left it as a TODO